docs/src/main/sphinx/connector/kudu.rst

```diff
@@ -15,6 +15,14 @@ The Kudu connector is compatible with all Apache Kudu versions starting from 1.0
 If the connector uses features that are not available on the target server, an error is returned.
 Apache Kudu 1.8.0 is currently used for testing.
```

Comment:
Can we remove this statement (version used for testing) since this is guaranteed to go out of date and it's implicitly covered by the requirements section now?

Reply:
Done.